data-table/__tests__/table.js

```diff
@@ -58,6 +162,27 @@ describe('API basic tests', () => {
         expect(checkboxList.length).toEqual(9);
     });
 
+    test('properly create checkbox field within each cell', () => {
+        let dropdownList = cellFieldList('cell-dropdown');
+        for (let i = 0; i < 9; i++) {
+            let res = dropdownList[i].options[0].text;
+            expect(res).toBe("Active");
+        }
+    });
+
+    test('properly create checkbox field within each cell', () => {
+        let dropdownList = cellFieldList('cell-dropdown');
+        for (let i = 0; i < 9; i++) {
+            let res = dropdownList[i].options[1].text;
+            expect(res).toBe("Inactive");
```

Comment:
This test is sufficiently similar to the previous one that you could consider combining them, or removing this test entirely. Each test should test unique functionality, so you may consider testing (1) that the first option is "Active," and (2) that the length of the options is 2.